doc/source/whatsnew/v0.21.0.txt

```diff
@@ -49,6 +49,7 @@ Backwards incompatible API changes
 
 - Accessing a non-existent attribute on a closed :class:`HDFStore` will now
   raise an ``AttributeError`` rather than a ``ClosedFileError`` (:issue:`16301`)
+- ``pd.read_csv()`` now treats ``'n/a'`` strings as missing values by default (:issue:`16078`)
```

Comment:
I could, but @jorisvandenbossche made exactly the opposite request: https://github.com/pandas-dev/pandas/pull/16079#issuecomment-297853157